go.mod

```diff
@@ -118,3 +118,19 @@ require (
 	golang.org/x/text v0.13.0 // indirect
 	golang.org/x/tools v0.12.0 // indirect
 )
+
+replace github.com/donomii/go-rwkv.cpp => /home/cryptk/Documents/sourcecode/LocalAI/sources/go-rwkv
```

Comment:
no need to commit this changeset, probably this is breaking the build

Reply:
Doh, I should update make clean to remove the replacements

Reply:
All fixed, I also added a dropreplace Make target that removes those replacements and added it to the clean target